Examples/Algorithms/Geant4/src/Geant4Simulation.cpp

```diff
@@ -130,6 +130,12 @@ ActsExamples::ProcessCode ActsExamples::Geant4Simulation::execute(
   // Set the seed new per event, so that we get reproducible results
   G4Random::setTheSeed(m_cfg.randomNumbers->generateSeed(ctx));
 
+  // Set the seed new per event, so that we get reproducible results
+  G4Random::setTheSeed(m_cfg.randomNumbers->generateSeed(ctx));
+
+  // Set the seed new per event, so that we get reproducible results
+  G4Random::setTheSeed(m_cfg.randomNumbers->generateSeed(ctx));
+
```

Comment:
unintentional duplication?

Reply:
yeah that should not be here actually. reminder from a earlier PR